test/test.js

```diff
@@ -0,0 +1,17 @@
+var request = require('request'),
+		assert = require('assert'),
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/test/test.js
+++ b/test/test.js
@@ -1,5 +1,5 @@
 var request = require('request'),
-		assert = require('assert'),
+        assert = require('assert'),
 		base_url = "http://localhost:3000/";
 
 describe("Job portal Index",function(){
```